src/Core/Tools/Entities/Send.cs

```diff
@@ -1,29 +1,123 @@
+    /// </remarks>
     public Guid? OrganizationId { get; set; }
+
+    /// <summary>
+    /// Describes the data being sent. This field determines how
+    /// the <see cref="Data"/> field is interpreted.
+    /// </summary>
     public SendType Type { get; set; }
-    public string Data { get; set; }
-    public string Key { get; set; }
+
+    /// <summary>
+    /// Stores data containing or pointing to the transmitted secret. JSON.
+    /// </summary>
+    /// <note>
+    /// Must be nullable due to several database column configuration.
+    /// The application and all other databases assume this is not nullable.
+    /// Tech debt ticket: PM-4128
+    /// </note>
+    public string? Data { get; set; }
+
+    /// <summary>
+    /// Stores the data's encryption key. Encrypted.
+    /// </summary>
+    /// <note>
+    /// Must be nullable due to MySql database column configuration.
+    /// The application and all other databases assume this is not nullable.
+    /// Tech debt ticket: PM-4128
+    /// </note>
+    public string? Key { get; set; }
+
+    /// <summary>
+    /// Password provided by the user. Protected with pbkdf2.
+    /// </summary>
     [MaxLength(300)]
-    public string Password { get; set; }
+    public string? Password { get; set; }
+
+    /// <summary>
+    /// The send becomes unavailable to API callers when 
+    /// <see cref="AccessCount"/>  &gt;= <see cref="MaxAccessCount"/>.
+    /// </summary>
     public int? MaxAccessCount { get; set; }
+
+    /// <summary>
+    /// Number of times the content was accessed.
+    /// </summary>
+    /// <remarks>
+    /// This value is owned by the server. Clients cannot alter it.
+    /// </remarks>
     public int AccessCount { get; set; }
+
+    /// <summary>
+    /// The date this send was created.
+    /// </summary>
     public DateTime CreationDate { get; internal set; } = DateTime.UtcNow;
+
+    /// <summary>
+    /// The date this send was last modified.
+    /// </summary>
     public DateTime RevisionDate { get; internal set; } = DateTime.UtcNow;
+
+    /// <summary>
+    /// The date this send becomes unavailable to API callers.
+    /// </summary>
     public DateTime? ExpirationDate { get; set; }
+
+    /// <summary>
+    /// The date this send will be unconditionally deleted.
+    /// </summary>
+    /// <remarks>
+    /// This is set by server-side when the user doesn't specify a deletion date.
+    /// </remarks>
     public DateTime DeletionDate { get; set; }
+
+    /// <summary>
+    /// When this is true the send is not available to API callers,
+    /// unless they're the creator.
+    /// </summary>
     public bool Disabled { get; set; }
+
+    /// <summary>
+    /// When this is false the creator's email address is not shown to the recipient.
+    /// </summary>
+    /// <remarks>
+    /// This should is null for compatibility with clients that don't set the field.
+    /// ::ADD TECH DEBT TICKET TO REMOVE NULLABILITY::
```

Comment:
nit: Is this meant to be `This field is null`? Also, maybe missing a ticket ID. 

It might be nice to include in the remarks what true boolean value should be defaulted to in the case where this is null. I'm guessing `false` but I'll leave the exacts to you.

Reply:
That comment was notes from a conversation that I forgot to update. Will clean it up presently! Thanks! :)